doc/changelog.md

```diff
@@ -75,6 +75,9 @@
   the current state-vector before qubit re-allocations.
   [(#479)](https://github.com/PennyLaneAI/catalyst/pull/479)
 
+* Add support for post-selection in mid-circuit measurements.
+  [(#491)](https://github.com/PennyLaneAI/catalyst/pull/491)
```

Comment:
You could add the example in the changelog

Reply:
Done